src/main/java/frc/team2412/robot/commands/intake/AllOutCommand.java

```diff
@@ -0,0 +1,25 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.team2412.robot.subsystems.IntakeSubsystem;
+
+public class AllOutCommand extends Command {
```

Comment:
Out seems ambiguous.  It might spit the note out the bottom or out the launcher.